tests/unit/libbpf_test.cpp

```diff
@@ -1005,6 +1005,11 @@ TEST_CASE("libbpf map", "[libbpf]")
     REQUIRE(result < 0);
     REQUIRE(errno == EINVAL);
 
+    // next_key is NULL.
+    result = bpf_map_get_next_key(map_fd, NULL, &value);
```

Comment:
question: the second argument to `bpf_map_get_next_key()` is `previous_key`, which is being passed as NULL here, whereas the comment says `next_key` is NULL. The above code change is also for handling `next_key` being NULL. Is this expected?

Reply:
Thanks, good catch!